### ISSUE TITLE
Rule 301: exclude commands only checking syntax/configuration 

### ISSUE THREAD

**Issue by simon04** [ansible/ansible-lint#704]
# Issue Type
- [x] Bug report
- [x] Feature request

# Ansible and Ansible Lint details

```
$ ansible --version
ansible 2.9.6
$ ansible-lint --version
ansible-lint 4.2.0
```

OS package via Arch Linux

# Desired Behaviour

I would like to test the nginx configuration on the server using `nginx -t`. This should not trigger rule 301.

# Actual Behaviour (Bug report only)

The following snippet triggers rule 301:

```yml
---
- name: Check nginx config
  command: nginx -t -c /etc/nginx/nginx.conf
```

**Comment by webknjaz** [comment]
We're not going to special-case every command possible. The proper way of using them is to wrap those commands as Ansible modules. In ansible-lint you have an option to ignore a line with `  # noqa` comments. But the rule is good as it is.
